sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Utils.java

```diff
@@ -101,6 +105,17 @@ private static ObjectMapper createAndInitializeObjectMapper(boolean allowDuplica
         return objectMapper;
     }
 
+    private static ObjectMapper createAndInitializeDurationObjectMapper() {
+        ObjectMapper objectMapper = new ObjectMapper();
+        SimpleFilterProvider filterProvider = new SimpleFilterProvider();
+        objectMapper.registerModule(new SimpleModule()
+                .addSerializer(Duration.class, ToStringSerializer.instance)
+                .addDeserializer(Duration.class, DurationDeserializer.INSTANCE)
+                .addSerializer(Instant.class, ToStringSerializer.instance))
+            .setFilterProvider(filterProvider);
```

Comment:
why do we need the filter provider?

Reply:
Just curious - the deserializer for `Duration` - is this needed for a `objectMapper` primarily used in `toString()`? If so, why no deserializer for `Instant`?

Reply:
These are the configs used in RntbdObjectMapper. Serialization works as expected without filter provider and deserializer settings as well. So I've removed them.